fi.laamaa.m8c.yaml

```diff
@@ -0,0 +1,52 @@
+app-id: fi.laamaa.m8c
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: m8c
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland? If yes then it's `fallback-x11` on the preceding line. other remove the wayland second.

(and I don't mean Xwayland, which is x11)